src/components/Autocomplete/styles.scss

```diff
@@ -49,6 +49,9 @@
     height: '21px';
   };
   & .MuiOutlinedInput-root {
+    fieldset {
```

Comment:
definitely simpler 🎉 thanks!!
i think we may need a similar change for the error condition.
if it helps, you can play around in storybook (the error text is settable in the ui)

Reply:
Yes I missed the error border on this one, thanks!

Reply:
Looks good 👍 thanks!